src/python/finalize/finalize.go

```diff
@@ -96,6 +96,14 @@ func (f *Finalizer) HandleCollectstatic() error {
 	return nil
 }
 
+func hasDjangoAsDependency(f *Finalizer) error {
+	if err := f.Command.Execute(f.Stager.BuildDir(), os.Stdout, os.Stderr, "pip-grep", "-s", "requirements.txt", "django", "Django"); err != nil {
+		return f.Command.Execute(f.Stager.BuildDir(), os.Stdout, os.Stderr, "grep", "-i", "Django", "-q", "-S", "Pipfile")
```

Comment:
I'm unable to find a `-S` flag for grep in the man page. Please let me know if I'm wrong.
Also, wouldn't simply grep-ing for "django" match if the [[source]] uri has django in it, or if it uses a package/dev-package like ng2django? Could you please parse the Pipfile instead of grep-ing?

Reply:
Agreed. I think something stricter might be better here. Perhaps even a well-crafted regex could suffice (with accompanying test cases).